docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  ioc:
+    image: ghcr.io/lnls-dig/ipmi-mgr-epics-ioc
+    build:
+      context: ./
+      dockerfile: docker/Dockerfile
+      target: static-link
+      labels:
+        org.opencontainers.image.source: https://github.com/lnls-dig/ipmi-mgr-epics-ioc
+      args:
+        REPONAME: ipmi-mgr-epics-ioc
+        RUNDIR: /opt/ipmi-mgr-epics-ioc/iocBoot/iocIpmiMgr
+          #ENTRYPOINT: ./YOUR_ENTRYPOINT
+    command: ./runIpmiMgr.sh -P HOMOLOG -R DE22 -i de-22rabpm-co-cratectrl
```

Comment:
Leftover command.

And usually we don't leave command with something to be used for debugging, but I don't know if it's a bad idea...

I think we should cleanup the scripts used here, though. The containers for the machine usually require a complicated shell script mess, when they could just use environment variables in the docker compose file, which have the advantage of being self-documenting (we can discuss if that pattern makes more sense in an IOC meeting); then, it also allows to directly use `lnls-run`.

Reply:
Do you think we may follow something similar to what we did for RFFE IOC?

That is, https://github.com/lnls-dig/rffe-epics-ioc/commit/655c72f921ed841c9aaae89ee8d1dfd908140073 (simplified by https://github.com/lnls-dig/rffe-epics-ioc/commit/286286f6c084c1ac31949e48f714d631886fafe2).

Reply:
I don't think so, unless all you want is an example config... The actual deployment will be in the internal docker-stacks repository, since this service will run on the cluster and not on the crates' CPUs (we want to be able to remotely restart the CPU)